ui/app/AppLayouts/Wallet/views/TokenSelectorAssetDelegate.qml

```diff
@@ -17,6 +17,7 @@ ItemDelegate {
     required property string symbol
     required property string currencyBalanceAsString
     required property string iconSource
+    required property bool isFirstSearchEntry
```

Comment:
This property seems not necessary as we already have `highlighted` property there which could be used to highlight first element if necessary.

Currently we can highlight even 3 items at once, what doesn't seem to be correct:

![Screenshot from 2024-11-18 13-11-01](https://github.com/user-attachments/assets/f2da7989-6e26-4544-bb3a-dd5b4e090610)

(first, selected and hovered)

Setting `highlighted in `SearchableAssetsPanel` seems to solve it nicely:
```qml
                highlighted: !!searchBox.text
                             ? (index === 0)
                             : (model.tokensKey === root.highlightedKey)
```

Reply:
but perhaps the name could be more generic ?

Reply:
But isn't the situation on the screenshot a bit strange?

Maybe if we want to have the first item "hovered" by default, that hover should disappear when we explicitly hover over another item? And go back to first when we are completely outside with the mouse cursor?

Reply:
aaah I dont know but I overlooked that issue. Yes you are right, should be hovered by default but then should change if the user manually hovers over anther item.

Reply:
@micieslak updated property name and behaviour now

https://github.com/user-attachments/assets/5ca63760-83e9-4414-bda1-924f4ef24354